pandas/core/series.py

```diff
@@ -5745,7 +5745,50 @@ def to_period(self, freq: str | None = None, copy: bool | None = None) -> Series
     _info_axis_name: Literal["index"] = "index"
 
     index = properties.AxisProperty(
-        axis=0, doc="The index (axis labels) of the Series."
+        axis=0, doc="""The index (axis labels) of the Series.
+           
+            Get the index (row labels) of the Series.
+
+            Returns
+            -------
+            Index
+                The index labels of the Series.
+
+            See Also
+            --------
+            Series.reindex: 
+                Conform Series to new index.
+            Series.set_index: 
+                Set Series as DataFrame index.
+            Index: 
```

Comment:
https://github.com/pandas-dev/pandas/actions/runs/4584805993/jobs/8096531909?pr=51842#step:4:286

You have included trailing whitespace here. It is useful to show whitespace characters in your editor to avoid this.

```suggestion
            Series.reindex:
                Conform Series to new index.
            Series.set_index:
                Set Series as DataFrame index.
            Index:
```